pkg/commands/commands_test.go

```diff
@@ -83,25 +87,34 @@ fi
 if [ ! -d ${RESULTS_DIR_PATH} ]; then
   mkdir -p ${RESULTS_DIR_PATH}
 fi
-gatling.sh -sf ${SIMULATIONS_DIR_PATH} -s testSimulationClass -rsf ${RESOURCES_DIR_PATH} -rf ${RESULTS_DIR_PATH} 
 
-if [ $? -ne 0 ]; then
+if [ ${SIMULATIONS_FORMAT} = "bundle" ]; then
+  gatling.sh -sf ${SIMULATIONS_DIR_PATH} -s ${SIMULATION_CLASS} -rsf ${RESOURCES_DIR_PATH} -rf ${RESULTS_DIR_PATH}  -rm local
+elif [ ${SIMULATIONS_FORMAT} = "gradle" ]; then
+  ./gradlew -Dgatling.core.directory.results=${RESULTS_DIR_PATH} gatlingRun-${SIMULATION_CLASS} 
```

Comment:
Is there a reason why you don't pass the simulations or resources directories as arguments?

Reply:
with [Gatling Gradle Plugin](https://docs.gatling.io/reference/extensions/build-tools/gradle-plugin/), these files do have quite typical locations: 
- simulations are under `src/gatling/java` or `src/gatling/kotlin` or `src/gatling/java`
- resources are under `src/gatling/resources`

I see that there's an option to customize these options via:
```
sourceSets {
  gatling {
    scala.srcDir "folder1" <1>
    // or
    scala.srcDirs = ["folder1"] <2>

    resources.srcDir "folder2" <3>
    // or
    resources.srcDirs = ["folder2"] <4>
  }
}
```
but I do not know upfront in which language the simulations were written, so it's quite hard to guess. IMHO convention over configuration rule applies here perfectly :)